containers/how-to-create-singularity-image-from-local-docker-image.txt

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
If this is a `.txt` why the `bash` interpreter?   Also, this reads more like a script than a how-to.